test/b2gedmntuples_cfg.py

```diff
@@ -749,9 +749,24 @@
     candSVTagInfos         = cms.string("pfInclusiveSecondaryVertexFinder"), 
 )
 
+
+process.photonJetsUserData = cms.EDProducer(
+    'PhotonJets',
+    phoLabel = cms.InputTag("slimmedPhotons"),
+    pv        = cms.InputTag(pvLabel),
+    rho               = cms.InputTag(rhoLabel),
+    packedPFCands = cms.InputTag("packedPFCandidates"),
+    jetLabel  = cms.InputTag("jetUserDataAK8"),
+    ebReducedRecHitCollection = cms.InputTag("reducedEgamma:reducedEBRecHits"),
+    eeReducedRecHitCollection = cms.InputTag("reducedEgamma:reducedEERecHits")
+
+    )
+
+
+
 process.boostedJetUserDataAK8 = cms.EDProducer(
     'BoostedJetToolboxUserData',
-    jetLabel  = cms.InputTag('jetUserDataAK8'),
+    jetLabel  = cms.InputTag('photonJetsUserData'),
     #topjetLabel = cms.InputTag('patJetsCMSTopTagCHSPacked'),
```

Comment:
The photonJet collection has some variables for the case when you have photons inside jets. 
He changed this here, because before these quantities were separated from the jets, and 
 analyzers need to matched them to the jets in their analyzers. Now, Savvas is just adding those quantities as userFloats of the jets for simplicity (and to not to change the label of all the modules, he add the variables before the boostedJetUserData).